docs/src/Atmos/Model/risingbubble.md

```diff
@@ -0,0 +1,447 @@
+        linear_model = AtmosAcousticGravityLinearModel,
+        slow_method = LSRK144NiegemannDiehlBusch,
+        fast_method = LSRK144NiegemannDiehlBusch,
+        timestep_ratio = 10,
+    );
+nothing #hide
+```
+
+Since we want four tracers, we specify this and include
+the appropriate diffusivity scaling coefficients (normally these
+would be physically informed but for this demonstration we use
+integers corresponding to the tracer index identifier)
+
+```@example risingbubble
+    ntracers = 4
+    δ_χ = SVector{ntracers, FT}(1, 2, 3, 4)
+```
+
+A model coefficient for the turbulence closure is defined,
+
+```@example risingbubble
+    C_smag = FT(0.23)
+```
+
+A reference state for the linearisation step is also defined.
+
+```@example risingbubble
+    ref_state =
+        HydrostaticState(DryAdiabaticProfile(typemin(FT), FT(300)), FT(0))
+```
+
+The fun part ! Here we assemble the `AtmosModel`.
+
+```@example risingbubble
+    model = AtmosModel{FT}(
+        AtmosLESConfigType, # Flow in a box, requires the AtmosLESConfigType
+        param_set; # [Parameter](@ref paramset) set corresponding to earth parameters
+        turbulence = SmagorinskyLilly(C_smag), # Turbulence closure model
+        hyperdiffusion = StandardHyperDiffusion(60), # Hyperdiffusion (4th order) model
+        source = (Gravity(),), # Gravity is the only source term here
+        tracers = NTracers{ntracers, FT}(δ_χ), # Tracer model with diffusivity coefficients
+        ref_state = ref_state, # Reference state
+        init_state = init_risingbubble!, # Apply the [initial condition](@ref init)
+    )
+```
+
+!!! note
+    The `AtmosModel` type allows sensible default values, and you can leave fields
+    for which you wish to use the defaults out of the model specification.
+    At the very least you must specify the initial conditions. Detailed information
+    on subcomponent argument options can be found in their respective docs.
+
+Finally,  we pass a `Problem Name` string, the mesh information, and the model type to  the `AtmosLESConfiguration` object.
+
+```@example risingbubble
+    config = CLIMA.AtmosLESConfiguration(
+        "DryRisingBubble", # Problem title [String]
+        N,                 # Polynomial order [Int]
+        resolution,        # (Δx, Δy, Δz) effective resolution [m]
+        xmax,              # Domain maximum size [m]
+        ymax,              # Domain maximum size [m]
+        zmax,              # Domain maximum size [m]
+        param_set,         # Parameter set. Model optional but parameter set not optional.
+        init_risingbubble!,# Function specifying initial condition
+        solver_type = ode_solver, # Time-integrator type
+        model = model, # Model type
+    )
+    return config
+end
+```
+
+!!! note
+    `Keywords` are used to specify some arguments (see appropriate source files).
+
+## Diagnostics
+
+```@example risingbubble
+function config_diagnostics(driver_config)
+    interval = "10000steps"
```

Comment:
Just a random comment, are we really parsing a string here? That seems more error prone than it needs to be. 

Is the point here that we could use time passed in `s` or simulation steps? If so we might want to consider punning on Unitful and allowing `1000s` and `1000steps`

Reply:
Yes, we're parsing that string ([here](https://github.com/climate-machine/CLIMA/blob/master/src/Utilities/Callbacks/Callbacks.jl#L358-L403)) as it could also come from the command line.